tasks/db_deploy/install/create_db.py

```diff
@@ -203,3 +206,60 @@ def create_inventory_objects(connection: Connection) -> None:
     logger.debug("Creating files table ...")
     connection.execute(sql.files_table_sql())
     logger.info("files table created.")
+
+
+def create_internal_reconciliation_objects(connection: Connection) -> None:
+    """
+    Creates the ORCA internal reconciliation tables in the proper order.
+    - reconcile_status
+    - reconcile_job
+    - reconcile_s3_object
+    - reconcile_catalog_mismatch_report
+    - reconcile_orphan_report
+    - reconcile_phantom_report
+
+    Args:
+        connection (sqlalchemy.future.Connection): Database connection.
+
+    Returns:
+        None
+    """
+    # Create reconcile_status table
+    logger.debug("Creating reconcile_status table ...")
+    connection.execute(sql.reconcile_status_table_sql())
+    logger.info("reconcile_status table created.")
+
+    # Create reconcile_job table
+    logger.debug("Creating reconcile_job table ...")
+    connection.execute(sql.reconcile_job_table_sql())
+    logger.info("reconcile_job table created.")
+
+    # Create reconcile_s3_object table
+    logger.debug("Creating reconcile_s3_object table ...")
+    connection.execute(sql.reconcile_s3_object_table_sql())
+    logger.info("reconcile_s3_object table created.")
+
+    # Create reconcile_catalog_mismatch_report table
+    logger.debug("Creating reconcile_catalog_mismatch_report table ...")
+    connection.execute(sql.reconcile_catalog_mismatch_report_table_sql())
+    logger.info("reconcile_catalog_mismatch_report table created.")
+
+    # Create reconcile_orphan_report table
+    logger.debug("Creating reconcile_orphan_report table ...")
+    connection.execute(sql.reconcile_orphan_report_table_sql())
+    logger.info("reconcile_orphan_report table created.")
+
+    # Create reconcile_phantom_report table
+    logger.debug("Creating reconcile_phantom_report table ...")
+    connection.execute(sql.reconcile_phantom_report_table_sql())
+    logger.info("reconcile_phantom_report table created.")
+
+    # Create partition table
+    logger.debug("Creating partition table for reconcile_s3_object ...")
+    connection.execute(sql.reconcile_s3_object_partition_sql())
```

Comment:
Missing partition name and loop for multiple partitions.

Reply:
Also missing bucket_name

Reply:
how do we get the list of partition_names? Is this how we are thinking to do this?  
```
# Create partition table
    logger.debug("Creating partition table for reconcile_s3_object ...")
    partition_names =[]
    for partition_name in partition_names:
        connection.execute(sql.reconcile_s3_object_partition_sql(partition_name))
        logger.info("partition table for reconcile_s3_object created.")

```

Reply:
This is the thing we talked about yesterday where you call a function in shared_reconciliation with each orca bucket name. See get_partition_name_from_bucket_name in shared_reconciliation.py in my pr.

Reply:
Updated logic and calls to be able to create multiple partitions.